pkgs/sdk_triage_bot/tool/bench.md

```diff
@@ -0,0 +1,110 @@
+## What's this?
+
+This is a benchmark of existing, previously triaged issues. We use it to
+measure changes to our triage classification prompt - to see if changes
+materially improve the classification performance.
+
+## General issues
+
+| Issue | Expected |
+| --- | --- |
+| #56366 | `area-dart2wasm`, `type-enhancement`            |
```

Comment:
Maybe use more than 100 issues? Should be easy enough given the number of SDK issues, and would help against overtraining (of us in this case :) ).

Reply:
👍 generally. These were selected so that the number of each `area-` group roughly corresponds to how they're represented in the issue tracker. I then manually went through the ones that didn't agree with the prediction to see if they're items we actually want in the benchmark (too odd to benchmark, over-represented, ...).

So, we should add more over time, it's just a bit of a manual process.

Reply:
I thought this was a generated list - then nevermind :D